src/Distribution/Server/Pages/Group.hs

```diff
@@ -73,3 +74,8 @@ listGroup [] _ = p << "No member exist presently"
 listGroup users muri = unordList (map displayName users)
   where displayName uname = (anchor ! [href $ "/user/" ++ display uname] << display uname) +++
                             maybe [] (removeUser uname) muri
+
+listGroupCompact :: [Users.UserName] -> Html
+listGroupCompact [] = toHtml "No member exist presently"
+listGroupCompact users = foldr1 (\a b -> a +++ ", " +++ b) (map displayName users)
+  where displayName uname = (anchor ! [href $ "/user/" ++ display uname] << display uname)
```

Comment:
The last line has copy-and-paste code from above.  Please factor out the common code.